superset/common/query_context_processor.py

```diff
@@ -307,6 +324,20 @@ def _get_timestamp_format(
 
         return df
 
+    def get_time_grain(
```

Comment:
The joys of Superset and its N different ways to store/retrieve the same information :facepalm: